### ISSUE TITLE
Credit card brand in the paymentMethod data

### ISSUE THREAD

**Issue by RyanLinXiang** [Adyen/adyen-react-native#207]
**Is your feature request related to a problem? Please describe.**
Our business needs to know how may payments have been selected for specific brands. Currently the `onSubmit` listener only provides payment type 'scheme' for credit cards.

**Describe the solution you'd like**
We would need a property like `brand` in the paymentMethod data to have the exact credit card brand name.

**Describe alternatives you've considered**
No workarounds found so far.

**Comment by descorp** [comment]
Hey @RyanLinXiang 

Thanks for reaching out 💚

Currently it is possible on iOS, but not on Android. 
There is #210 PR for Android quick fix.

**Comment by RyanLinXiang** [comment]
Hey! Very nice to know - that is exactly what we need. I hope that PR is going to be merged soon. 

Thank you so much!

**Comment by RyanLinXiang** [comment]
I tested PR #210 on my end and in Android, `onSubmit` still does not include the `brand` key. Also on iOS I could not see any `brand` key. See `paymentMethodData` passed into `onSubmit` handler attached below.

For testing I used the following test Visa card:

`4111 1111 4555 1142`, `03/30`, `737`

```
{
  "browserInfo": {
    "userAgent": "xxx"
  },
  "paymentMethod": {
    "encryptedCardNumber": "eyJlbmMiOiJBMjU2Q0JDLUhTNTEyIiwiYWxnIjoiUlNBLU9BRVAtMjU2IiwidmVyc2lvbiI6IjEifQ.Qgp2_NnY9Exe8rO_bPsDTVQgSA8VqlcDbLSQBob22Qd24eGDcyQPx8dlJSDqZIZLk7Ie3XORGFD8Rv-VYSOx-drqsjb-XXgMVuXq9RS8dY3tl-WuK5kRJbisfEhsgzRh6uIqPQ6wFF1ECXcXG5-1pItzgx3yD_ZtY-Ax2ILETMjfJoUAa70Ntu3g5-6P_zViMGJNJoHZw5jx5b7walBIman0wP-9UeVxiIdLPhJmHSpNvln5SLhqXEMaKWiygiFbLbjn1GYT5H4TPf6VETnTYOYjj_3m1iWrrNUlaLb6y0abd1t6kidgt2hYfc9qyvm8_ojokZBNpfA25avIOdWTSA.y862vnufndvnPyrW2bwa9A.VYy8WVn3qJ966a3vMFrjTqo5ae_k7InzWeF9dPzUHEKky-6gJsDbJT3GueqkBNxUiG1aK9nfTYrBKkJQV7Xsomq7AeKJNC3DDE_puT39nh0.Vm9KuCfWstWDnhNZPYb2C3N1n3WeKnSUu579dfq5D8k",
    "encryptedExpiryMonth": "eyJlbmMiOiJBMjU2Q0JDLUhTNTEyIiwiYWxnIjoiUlNBLU9BRVAtMjU2IiwidmVyc2lvbiI6IjEifQ.eVtQjD00AOiYCQp5s6Qmu0FCBrnQs6qXhOhXBVfWpJnwX_F8HEvzutq0ETLf_pTKqk8DzyXPEcSHo8zhs6V5W0Rf5LGeDdUr4zwORaFxYKdAPkyKjQrPIVPEPX6oDVUWN06mOYYQT_1yTemlAUNk0jz2dft48jTFmi7u3PiQwG8sSuEHQusHIxcbZ293vTXtLw28KWv5Enh4XqE9mbTqsNuGHXLLewfDkRhwgVoURK_S4ZcYsJzcPNeoY4cMsyl_waTbtOGCU59cYaoM-n-bH4xhYMZht7Ck5-MGe6UcPppiD-L_56tPzd65jeribCONnmvhr0SFNiv0KR5Lf7Vo2A.YWp1oZNuMExyYAu9y8Kw4w.zD8mBei_fnEGlSX5BZP8qErYh-h41FwE1_j_vVmDoU_y79baDiojpKRYzXbT6u0WAEEXWr9Ou62tLoi4LZxfyw.5AQcbkA7LCh62Z9vo5gY3B0-ATrWF1QRup3ulU1PPMo",
    "encryptedExpiryYear": "eyJlbmMiOiJBMjU2Q0JDLUhTNTEyIiwiYWxnIjoiUlNBLU9BRVAtMjU2IiwidmVyc2lvbiI6IjEifQ.H9erHsYBt82GlwxqNeVVhMBtCHXP-t64WU68UcRGrjS4iY7JSUjnea3DcCBXceRScSSSKo5bEuTrcF39-mrojYQR1QqWQt_F2d0UI95vwrzN7PwTIuVj3DVZgdbEXunpYMWoQezwujMKIyHtT8InpHAWPW2LQa6X_R60a--BEaHdD5-ybIWRdAfIhP_eTBj5XnqH5TdIPGuhpzPAU--4gWu4h64tgNEIYvPZSyFbZBuNkBLNBmBmRjZ0Cw5nM2f2iY1C4xcQq2hDWwE_PzttYmrujC8CWuLJObnJPJ48dYIZoH2CBWB_e_HzLMUDZBk1f6AcG0ZZCVUEFM9rHiylKw.VW2rOT9z7PKxXqbO-JnveA.w9huP-zdFYluu8Hj6WG2579mex-spw81qeRIvgBIdOLxyXZttxNJJab0rMZikBbcZkA1GKO6yZjb-OKPPJVP7g.RnRMKZilb6HkyWxIoB22ntAhi6Oakqysm4VI8Wcc0Zc",
    "encryptedSecurityCode": "eyJlbmMiOiJBMjU2Q0JDLUhTNTEyIiwiYWxnIjoiUlNBLU9BRVAtMjU2IiwidmVyc2lvbiI6IjEifQ.K1ebiYt5pihV55-N7S_7hncfKlnDVBK5cyL54MV7rnIYMU1q_4F2M-TAVcGtu34ebpz2TNawpjXatUhzgus1kBVG42LXuMpONu-YWm7yVVXiySyCV17rZCsMGxx7QmIx7Xr_pcn9Z1HOswpvstdN6NQyCrz9jMaTANiYXXyA8qCxRFqTTmMqlxRkexo803iv3ud6Gud18UoIBuI78vIFa65-Lhpz4vjvkwqAxW5JtdlGKruedlN05MSx180BMWphCdulNp5WTQPZGvT5K8-OKtMinHj3xoH3Jd3SIUnaOl9EGy5iULqXhSVmNesoLeja3KdpAcxZcfcbijBdBGUwAA.UjofurQJ9myQ3rjr-f1dKQ.UtEjIZt9ehAB4r5h2gyY2ugiGLHyUnpK4gOcOu9-qIqyccYTXNVMkvG00nyJNNJ3nP5s1SktilZGOOyfeqKKOQ.igmeUU43VpmTcdDMPAlQ3OTJQN-TWLxFSZvNA3SSJlY",
    "threeDS2SdkVersion": "2.2.6",
    "type": "scheme"
  },
  "returnUrl": "myapp://payment",
  "storePaymentMethod": false
}
```

**Comment by descorp** [comment]
Hey @RyanLinXiang 

As far as I can tell `4111 1111 4555 1142` is not present on TEST Bin Lookup table.
Could you try with `5454 5454 5454 5454` ?

**Comment by descorp** [comment]
Released on [1.0.0](https://github.com/Adyen/adyen-react-native/releases/tag/1.0.0) 

I am going to close this thread for now.
Feel free to continue here and provide your feedback
